src/OpenConext/ProfileBundle/Resources/views/MyServices/overview.html.twig

```diff
@@ -76,7 +76,13 @@
                                     {% endif %}
                                     <tr>
                                         <td title="{{ attributeUrn }}">
-                                            <strong>{{ ('profile.saml.attributes.' ~ attribute.attributeDefinition.name)|trans({}, 'saml') }}</strong>
+                                            {% set translatedAttribute = ('profile.saml.attributes.' ~ attribute.attributeDefinition.name)|trans({}, 'saml') %}
+
+                                            {% if translatedAttribute == 'profile.saml.attributes.' ~ attribute.attributeDefinition.name %}
```

Comment:
So... why did we prefix the attributes in the SAML translation domain with `profile.saml.attributes.`?